doc/quick-install.md

```diff
@@ -23,7 +23,8 @@ Finally on the same machine test that you can ping the FQDN with: `ping "$(hostn
 ### Add the Jitsi package repository
 ```sh
 echo 'deb https://download.jitsi.org stable/' >> /etc/apt/sources.list.d/jitsi-stable.list
-wget -qO -  https://download.jitsi.org/jitsi-key.gpg.key | apt-key add -
+wget -qO -  https://download.jitsi.org/jitsi-key.gpg.key | sudo E: This command can only be used by root.
```

Comment:
I guess you ddidn't intend to paste the error here.

Reply:
Uhm, no o.O Was sure i just added 'sudo' in front. Thanks.